EMS/core-bundle/src/Resources/views/user/permissions/permissions.html.twig

```diff
@@ -0,0 +1,138 @@
+                            </tr>
+                            {% for contentType in emsco_get_content_types() %}
+                                <tr class="text-{{ contentType.color }}">
+                                    <td class="text-center">{{ contentType.name }}</td>
+                                    <td class="text-center"><i class="{% if contentType.icon %}{{ contentType.icon }}{% else %}fa fa-book{% endif %}"></i></td>
+                                    {% for role in [contentType.roles.view, contentType.roles.create, contentType.roles.edit, contentType.roles.publish, contentType.roles.delete, contentType.roles.trash, contentType.roles.archive, contentType.roles.show_link_create, contentType.roles.show_link_search] %}
+                                        <td class="{{ html_classes({'text-muted': role == 'not-defined'}) }} ">
+                                            {{ role|trans }}
+                                        </td>
+                                    {% endfor %}
+                                    <td>
+                                        <div class="btn-group">
+                                            {% if contentTypeCounts[contentType.id] != 0 %}
+                                                {% include '@EMSCore/elements/get-button.html.twig' with {
+                                                    'url':  path('permissions.view', {'contentType': contentType.id}),
+                                                    'label': 'View',
+                                                    'icon': 'tv'
+                                                }%}
+                                            {% endif %}
+                                        </div>
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="box ">
+                <div class="box-header with-border">
+                    <i class="fa fa-group"></i>
+                    <h3 class="box-title">{{ 'view.user.permissions.titles.roles_hierarchy'|trans }}</h3>
+                </div>
+                <div class="box-body">
+                    <div class="table-responsive">
+                        <p>{{ 'view.user.permissions.titles.descriptions.roles_hierarchy_description'|trans }}</p>
+                        <table class="table table-condensed table-striped">
+                            <thead>
+                            <tr>
+                                <th>#</th>
+                                <th>{{ 'view.user.permissions.table_headers.role'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.description'|trans }}</th>
```

Comment:
The description here is useless. Those are just names and does not have meaning, drop it